tests/issues.rs

```diff
@@ -0,0 +1,11 @@
+use std::str::FromStr;
+
+use cron::Schedule;
+
+#[test]
+fn issue_86() {
```

Comment:
Let's use a verbose name so test output in the terminal is more self-explanatory:
```suggestion
// Issue #86
fn shorthand_must_match_whole_input() {
```

I'd prefer to keep this unit test next to the code it's testing instead of making a new file.